doc/syntax.rst

```diff
@@ -318,9 +318,10 @@ Format specifications for chrono duration and time point types as well as
 .. productionlist:: sf
    chrono_format_spec: [[`fill`]`align`][`width`]["." `precision`][`chrono_specs`]
    chrono_specs: [`chrono_specs`] `conversion_spec` | `chrono_specs` `literal_char`
-   conversion_spec: "%" [`modifier`] `chrono_type`
+   conversion_spec: "%" [`padding_modifier`] [`locale_modifier`] `chrono_type`
    literal_char: <a character other than '{', '}' or '%'>
-   modifier: "E" | "O"
+   locale_modifier: "E" | "O"
+   padding_modifier: "E" | "O"
```

Comment:
These need to be changed because right now they are the same as locale modifiers.

Reply:
Fixed!